recipes/pyessv/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyessv" %}
+{% set version = "0.9.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 043949af39a00c0c01ba69ad27709b9ca03086166bd5aa4e279264b9ed495b57
+
+build:
+  skip: True  # [win]
```

Comment:
What is the reason for disabling windows? Seems like this package could be noarch.

Reply:
Addressed. Thanks for the review! Feel free to merge when ready!

Reply:
If you want to restrict this to unix only, you can just add the __unix virtual package. Or what was the reason for including that in the first place?

Reply:
I think the reason is that I spend a lot of my time in Linux. Nothing stopping pyessv from working for Windows users!